powerhome/playbook
# [PLAY-52]Remove div wrapping Image kit

#### Screens

<img width="1775" alt="image" src="https://user-images.githubusercontent.com/84349455/151196481-19a3267c-1c1f-40bc-8df6-adf2445f7159.png">

<img width="1174" alt="image" src="https://user-images.githubusercontent.com/84349455/151196533-e8bf0b78-f7ad-4742-a72e-b01d0938c831.png">

<img width="1771" alt="image" src="https://user-images.githubusercontent.com/84349455/151196585-37163c9a-dc07-437b-8e94-028200704c73.png">

<img width="780" alt="image" src="https://user-images.githubusercontent.com/84349455/151196613-0ee4341f-1d0a-4615-bc11-557a1c75cfdc.png">

<img width="575" alt="image" src="https://user-images.githubusercontent.com/84349455/151196731-c5e999f7-f996-4c6c-804f-0df0d02b35f0.png">


#### Breaking Changes

Yes, the image kit came with a built in wrapper `<div>` so alpha testing is required to ensure that the styling of the pages using this kit are not altered.

#### Runway Ticket URL

[https://nitro.powerhrg.com/runway/backlog_items/PLAY-52](url)

#### How to test this

1. Start up a server for nitro-web
2. Create or view a react component that uses the Image kit
3. Check to make sure that the image fits on the page correctly and looks the way it is anticipated to look

#### Checklist:
